sage/google_analytics_aggregate/V1.1.0__configure_ga4_reports.sql

```diff
@@ -0,0 +1,53 @@
+USE ROLE google_analytics_aggregate_admin;
+USE SCHEMA google_analytics.public;
+
+CALL CONFIGURE_REPORT(
+    REPORT_NAME => 'DEMOGRAPHIC_LOCATION',
+    PROPERTY_ID => '311611973',
+    DIMENSIONS => 'date,hostName,pagePath,pagePathPlusQueryString,continent,subcontinent,country,region,city',
+    METRICS => 'totalUsers,activeUsers,newUsers,sessions,averageSessionDuration,engagedSessions,engagementRate',
```

Comment:
Is there a totalUsers aggregate metric like totalUsers28days or totalUsers90days?

Reply:
I added both `active7DayUsers` and `active28DayUsers` since we had space for both metrics.

Do you think we should add these user metrics to the *VIEWS reports? Right now we're only capturing these metrics based on location, but it could be useful to provide user metrics for pages regardless of location.

Reply:
Thanks @philerooski 

1. Let's add active7 and 28 day users to as many tables as possible in these GA4 reports
2. I agree, can you let me know of example values for "continent,subcontinent,country,region,city"

I'm thinking maybe just "continent,country" is good for this first iteration.  We can always add another report with "country,city" if that's desired

Reply:
This [article](https://www.bounteous.com/insights/2023/09/14/ip-addresses-and-google-analytics-4-what-you-should-know/) explores how VPNs affect GA4 location data. In summary:

* Users appear where the VPN exits. Example: a US user on a EU VPN tallies as EU.
* Accuracy degrades at finer levels. Country is usually reliable; city is much less so. VPNs worsen this.
* Accuracy depends on the [level of location identification](https://www.iplocation.net/geolocation-accuracy). A device's country is correct 95–99 percent of time; region (including U.S. states and Canadian Provinces) 55–80 percent of the time; and cities are correct 50–75 percent of the time.

Reply:
> I think as a first pass, we first do "continent and country" and when we present this, we can ask the analysts if they would like city and region.

I agree. Particularly with the degraded accuracy we can expect as we get more specific, best to just provide this data at the country level.